src/bidiMapper/utils/cdpServer.spec.ts

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
Please rename file to cdpClient.spec.ts.

Reply:
done